net/nat/ipv4_nat_entry.c

```diff
@@ -195,39 +204,47 @@ static void ipv4_nat_entry_delete(FAR ipv4_nat_entry_t *entry)
  ****************************************************************************/
 
 #if CONFIG_NET_NAT_ENTRY_RECLAIM_SEC > 0
+static void ipv4_nat_reclaim_entry_cb(FAR ipv4_nat_entry_t *entry,
+                                      FAR void *arg)
+{
+  int32_t current_time = *(int32_t *)arg;
```

Comment:
```suggestion
  int32_t current_time = *(FAR int32_t *)arg;
```
why not use clock_t?

Reply:
Done, I only want to save a second value after `TICK2SEC`, and never need to be 64-bit?

Reply:
I just confuse why `int32_t` is used for time value, it would be more reasonable to use clock_t for time value If there is no guarantee that the var is within the range of int32_t.

Reply:
I mean, the expiration is in "seconds", we won't reach 2^31 seconds to expire an entry. (And 32768/65536 seconds is not enough while 2^63 is too much)